src/olympia/addons/tests/test_serializers.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+import uuid
+
+from elasticsearch_dsl import Search
+from rest_framework.test import APIRequestFactory
+
+from olympia import amo
+from olympia.amo.helpers import absolutify
+from olympia.amo.tests import addon_factory, ESTestCase, TestCase
+from olympia.addons.indexers import AddonIndexer
+from olympia.addons.models import Addon
+from olympia.addons.serializers import AddonSerializer, ESAddonSerializer
+
+
+class _TestAddonSerializerOutputMixin(object):
```

Comment:
The "regular" test class is `TestAddonSerializerOutput`, so the mixin does not need to be tested on its own. It's just that I have to make a mixin to work around inheritance madness for the ES variant of the tests.

I'll rename it.